applications/SwimmingDEMApplication/python_scripts/cellular_flow/make_mesh_ethier_benchmark_analysis.py

```diff
@@ -1,5 +1,5 @@
 from KratosMultiphysics import Parameters
-import ethier_benchmark_analysis
+import cellular_flow.ethier_benchmark_analysis as ethier_benchmark_analysis
```

Comment:
don't you also need the full path here?

Reply:
Done in next push. However it's not in the tests, maybe it's deprecated.